base.css

```diff
@@ -1,10 +1,89 @@
-@import "intro.css";
-@import "index.css";
-
 * {
   box-sizing: border-box;
   margin: 0;
   padding: 0;
   font-size: 100%;
   font-family: "Roboto", sans-serif;
 }
+
+.gridLayout {
+  position: absolute;
+  min-width: 414px;
+  height: 100vh;
+  background-color: RGB(25, 30, 40);
+  display: flex;
+  flex-direction: column;
+}
+
+.navigationBar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  width: 400px;
+  height: 75px;
+  transition-duration: 5s;
+}
+
+.navigationBarToggle {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  background-color: rgba(200, 200, 200, 0.1);
+  width: 400px;
+  height: 75px;
+
+  transition-duration: 5s;
+}
+.mainArea {
+  width: auto;
+  display: flex;
+  flex-direction: column;
+  background-color: transparent;
+
+  margin-left: 2%;
+}
+
+.menuIcon {
+  float: right;
+  margin: 3%;
+  color: seashell;
+  font-size: 270%;
+  opacity: 70%;
+  transition-duration: 0.7s;
+  text-shadow: 1pt 1pt rgba(200, 200, 200, 0.4);
+}
+
+.menuIconToggle {
+  opacity: 0%;
+  transition-duration: 0.7s;
+}
+
+.navButton {
+  opacity: 0%;
```

Comment:
The problem with using `opacity` to hide these buttons is that they are still usable even if not shown. I would advise to either remove them from the DOM or have them completely slide in and out of the viewport

Reply:
Hi Marwin, i saw the problem already. I just left the menu in the alpha state since it is visually ( demostration purpose) okay and the basic functionality is given. The moment all core features are done i will go into a heavy revision with my code with rgards to
- BAM and clean code
- CSS ratios and responsivenes
- UI functions (append, remove items, removing eventlistener)

!!! Change planed in dedicated user story for UI responsiveness

Thank you